esp/esp/program/forms.py

```diff
@@ -50,16 +50,17 @@ def make_id_tuple(object_list):
 class ProgramCreationForm(BetterModelForm):
     """ Massive form for creating a new instance of a program. """
 
-    term = forms.SlugField(label='Term or year, in URL form (i.e. "2007_Fall")', widget=forms.TextInput(attrs={'size': '40'}))
-    term_friendly = forms.CharField(label='Term, in English (i.e. "Fall 07")', widget=forms.TextInput(attrs={'size': '40'}))
+    term = forms.SlugField(label='Term or year, in URL form (e.g. <tt>2007_Fall</tt>)', widget=forms.TextInput(attrs={'size': '40'}))
+    term_friendly = forms.CharField(label='Term, in English (e.g. <tt>Fall 2007</tt>)', widget=forms.TextInput(attrs={'size': '40'}))
 
     teacher_reg_start = forms.DateTimeField(widget = DateTimeWidget())
     teacher_reg_end   = forms.DateTimeField(widget = DateTimeWidget())
     student_reg_start = forms.DateTimeField(widget = DateTimeWidget())
     student_reg_end   = forms.DateTimeField(widget = DateTimeWidget())
-    base_cost         = forms.IntegerField( label = 'Cost of Program Admission $', min_value = 0 )
-    sibling_discount  = forms.DecimalField(max_digits=9, decimal_places=2, required=False, initial=None, help_text='The amount of the sibling discount. Leave blank to disable sibling discounts.')
-    program_type      = forms.CharField(label = "Program Type")
+    base_cost         = forms.IntegerField(label = 'Cost of Program Admission $', min_value = 0 )
+    sibling_discount  = forms.DecimalField(max_digits=9, decimal_places=2, required=False, initial=None,
+                                                help_text='The amount of the sibling discount. Leave blank if you don\'t use sibling discounts.')
```

Comment:
Hmm, this changes the meaning -- is there in fact another setting that disables sibling discounts?

Reply:
Ok, this is fine then.  Instead of the `\'`, better to just use double-quotes for the string.